oxidation/crates/parsec_api_protocol/src/transport.rs

```diff
@@ -0,0 +1,69 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+use async_std::net::TcpStream;
+use async_tungstenite::{
+    accept_async, client_async, tungstenite,
+    tungstenite::{handshake::client::generate_key, Message},
+    WebSocketStream,
+};
+use futures::{
+    stream::{SplitSink, SplitStream},
+    SinkExt, StreamExt,
+};
+use http::{Request, Uri};
+
+pub struct Transport {
+    pub write: SplitSink<WebSocketStream<TcpStream>, Message>,
```

Comment:
I think we want write/read fields to be kept private

Reply:
Yes of course, I forgot this part after I switched branch and clippy disagreed

Reply:
Done :+1: